api/coreservice.go

```diff
@@ -930,7 +930,8 @@ func (core *coreService) Actions(start uint64, count uint64) ([]*iotexapi.Action
 
 	var res []*iotexapi.ActionInfo
 	var hit bool
-	for height := core.bc.TipHeight(); height >= 1 && count > 0; height-- {
+
+	for height := core.bc.TipHeight(); height >= 1; height-- {
```

Comment:
Because an error is returned when count==0 and totalactions-start is always greater than 0

Reply:
the init value of `count` may be larger than 0, but it will be updated in https://github.com/iotexproject/iotex-core/pull/3422/files#diff-a84e0d7fdb009bebf88eb3c2582dc4c2f5fce09f6d40a3a514b553c198675f42R908